cmd/krel/cmd/ff.go

```diff
@@ -77,11 +77,14 @@ func init() {
 
 func runFf(opts *Options) error {
 	// TODO: Add usage/help
-	// TODO: Check prerequisites (git, jq, go, make)
 	// TODO: Set positional args
 	// TODO: Fail on empty branch
 	// TODO: Fail on GITHUB_TOKEN not set
 
+	if !util.CommandsAvailable([]string{"git", "go", "make", "jq"}) {
```

Comment:
QQ here:
I thought `krel` uses go-git? Why do we have a dependency on git here too? Can we maybe only depend on either one?

Reply:
There are certain git commands that we use that are not supported in `go-git`.
Here's the compatibility list: https://github.com/src-d/go-git/blob/master/COMPATIBILITY.md